src/Management/ResourceBuilder.php

```diff
@@ -40,7 +40,7 @@ public function __construct()
     /**
      * @param array $data
      *
-     * @return Space|Asset|Upload|ContentType|Entry|EntrySnapshot|Locale|Webhook|WebhookCall|WebhookCallDetails|WebhookHealth|ResourceArray|PublishedContentType|User
+     * @return Space|Asset|Upload|ContentType|Entry|EntrySnapshot|Locale|Webhook|WebhookCall|WebhookCallDetails|WebhookHealth|ResourceArray|PublishedContentType|User|Organization
```

Comment:
Yeah, lot's of resources in this one. I could either leave it out and let the IDEs infer it (not sure what Sami will think of that) or I could mark it as `ResourceArray|ResourceInterface`.

WDYT?